dbt_project.yml

```diff
@@ -1,13 +1,14 @@
 name: 'shopify_source'
-version: '0.11.0'
+version: '0.12.0'
 config-version: 2
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 models:
   shopify_source:
     +schema: stg_shopify
-    +materialized: table
-    tmp:
-      +materialized: view
+    +materialized: view
```

Comment:
Opted for view over ephemeral since several downstream models require a view or table to run dbt_utils macros.